apis/apps/v1alpha1/apicast_types.go

```diff
@@ -166,6 +166,11 @@ type APIcastSpec struct {
 	// CustomPolicies specifies an array of defined custome policies to be loaded
 	// +optional
 	CustomPolicies []CustomPolicySpec `json:"customPolicies,omitempty"`
+
+	// ExtendedMetrics enables additional information on Prometheus metrics; some labels will be used with specific information that will provide more in-depth details about APIcast.
```

Comment:
Does this have an appreciable impact in performance(does it say so in apicast doc?) ? if so maybe it would be a good idea to document it

Reply:
@eloycoto question for you ^^